corehq/apps/hqadmin/management/commands/report_code_metrics.py

```diff
@@ -0,0 +1,45 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from dimagi.ext.couchdbkit import Document
+
+from corehq.util.metrics import metrics_gauge
+
+
+class Command(BaseCommand):
+    help = (
+        "Display a variety of code-quality metrics, optionally sending them to datadog. "
+        "Other metrics are computed in scripts/report-code-metrics.sh"
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--datadog',
+            action='store_true',
+            default=False,
+            help='Record these metrics in datadog',
+        )
+
+    def handle(self, **options):
+        self.datadog = options['datadog']
+        self.show_couch_docs_remaining()
+        self.show_custom_modules()
+
+    def show_couch_docs_remaining(self):
```

Comment:
nit: I realize this means remaining to migrate to SQL but that's not immediately obvious. When I first read the function name I thought it meant docs remaining in the DB after the tests have run.

```suggestion
    def show_couch_model_count(self):
```